content/fr/abstraction.md

```diff
@@ -5,7 +5,7 @@ category: Property
 tags: ["fundamental", "", ""]
```

Comment:
How about localizing tags? 😊
e.g., https://glossary.cncf.io/pt-br/tags/

Reply:
Indeed, I put the PR back to Draft to localize the tags.

Reply:
I just pushed a new commit to localize the tags